test/leader/leader_test.py

```diff
@@ -0,0 +1,74 @@
+from os import path
+
+import sys
+import unittest
+
+FILE_PATH = path.dirname(path.realpath(__file__))
+sys.path.append(FILE_PATH + "/../../src")
+
+from leader import Leader
+
+class LeaderTest(unittest.TestCase):
+    def test_set_speed_lower_bound(self):
+        leader = Leader()
+
+        self.assertEqual(leader.set_speed(-1), 0)
+
+
+    def test_set_speed_upper_bound(self):
+        leader = Leader()
+
+        self.assertEqual(leader.set_speed(1) , 50)
```

Comment:
set speed doesn't return anything.

Reply:
don't put a space between the parenthesis and the comma

Reply:
And why 50? What if I change the max speed?

Reply:
I followed the test case specification, the input and desired output was specified.

Reply:
instead of 50 i'll just put leader.MAX_SPEED